contentcuration/contentcuration/frontend/shared/data/serverSync.js

```diff
@@ -31,6 +31,10 @@ const SYNC_IF_NO_CHANGES_FOR = 2;
 // already instantiated in the broadcastChannel module.
 const channel = createChannel();
 
+// This is our local variable for determining whether we should
+// keep polling for changes and syncing them
+let keepPollingUnsyncedChanges = true;
```

Comment:
I think we can just change this to a tracker for the timeout? So this would just be:
```suggestion
let unsyncedPollingTimeoutId;
```